includes/backup.js

```diff
@@ -102,7 +108,7 @@ function validateBulkGetSupport(dbUrl, callback) {
  * to avoid enqueueing too many in one go.
  *
  * @param {string} log - log file name to maintain download state
- * @param {string} dbUrl - Source database URL
+ * @param {string} db - nodejs-cloudant db
  * @param {events.EventEmitter} ee - event emitter to emit received events on
  * @param {time} start - start time for backup process
```

Comment:
Unrelated to my changes, but yes it should. Fixed in 8cfff502b26d965df9c338652ba4648b460dc85c